.github/workflows/package.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065 # v5.6.0
         with:
-          python-version: "3.13"
+          python-version: "3.14.0-rc.2"
```

Comment:
Still a package noob sorry: shouldn't this be lowest version to maximize backwards compatibility?

Reply:
It shouldn't matter that much, but it also doesn't hurt, will update.